Application/EdFi.Ods.Api.Common/Dtos/AggregateRootWithCompositeKey.cs

```diff
@@ -0,0 +1,45 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using System.Runtime.Serialization;
+using EdFi.Ods.Common;
+
+namespace EdFi.Ods.Api.Common.Dtos
+{
+    public abstract class AggregateRootWithCompositeKey
+        : EntityWithCompositeKey,
+          IHasIdentifier,
+          IHasIdentifierSource,
+          IDateVersionedEntity
+    {
+        // -------------------------------------------------------------
+
+        // =============================================================
+        //                         Versioning
+        // -------------------------------------------------------------
+        [IgnoreDataMember] // Populate from etag header processing, don't serialize out or in.
+        public virtual DateTime LastModifiedDate { get; set; }
+
+        // nHibernate wraps property getter exception in PropertyAccessException if any
+        // underlying mapped properties are set to access "none", due to an invoke exception
+        // being triggered.  Placing the ChangeVersion here at the root to avoid this issue,
+        // and to make it available for future exposure if needed.
+        [IgnoreDataMember]
+        public virtual long ChangeVersion { get; set; }
```

Comment:
I know this is copied code, but this and CreatedByOwnershipTokenId should be documented as additional feature issues, we shouldn't have features creeping into the root objects.